### DISCUSSION TITLE
Import cost is very high of Daisyui in tailwind.config.js

### DISCUSSION THREAD

**Discussion by nagendragandla001** [saadeghi/daisyui#2929] [General]
<img width="513" alt="Screenshot 2024-03-22 at 4 00 16 PM" src="https://github.com/saadeghi/daisyui/assets/19658655/b758b43c-8188-4704-8286-67018645e555">

Daisy UI Version:  "daisyui": "^4.5.0",
Nextjs: 14
Import cost of the Daisyui is 381.7K. It is very high. With this my application's light house reports are failing. 

Any thoughts?

**Comment by saadeghi** [comment]
This is not related to lighthouse at all.  
Both Tailwind CSS and daisyUI are dev dependencies and they run in build time (not runtime) to generate a CSS file and if the config is correct, the CSS file size is expected to be small.  

This 381kb imported dependency is not going to be shipped to production.  
It's the size of all CSS in JS files that Tailwind CSS could use as source files.  
Unused styles are not going to be included in production, so Tailwind CSS generates a small CSS file and that small CSS file is going to be shipped in production site.

Let me know if you have any questions.
